tests/cli/test_cli.py

```diff
@@ -1230,6 +1230,18 @@ def test_upload_default_bundle_store(ctx):
     check_contains(bundle_store_name, _run_command([cl, "info", uuid]))
 
```

Comment:
We might want to add one more test where two nodes are spun up and we schedule two worker pods and make sure they both are scheduled on different nodes.

Reply:
Not sure how feasible this is, though.